### PR TITLE
BG-10752 support BSV

### PR DESCRIPTION
**Pull request by dannybitgo**
## Motivation
https://bitgoinc.atlassian.net/browse/BG-10752

## Overview Of Changes
Support BSV

### CODE REVIEW

File: app/sign.js [hunk 1]
```diff
@@ -128,7 +132,7 @@ const handleSignUtxo = function(recoveryRequest, key, skipConfirm) {
     console.log(`Signing input ${ i + 1 } of ${ recoveryRequest.inputs.length } with ${ derivedHDNode.neutered().toBase58() } (${ input.chainPath })`);
 
     // Handle BCH
-    if (recoveryRequest.coin === 'bch' || recoveryRequest.coin === 'tbch') {
+    if (recoveryRequest.coin === 'bch' || recoveryRequest.coin === 'tbch' || recoveryRequest.coin === 'bsv' || recoveryRequest.coin === 'tbsv') {
```
Comment (cooncesean) [inline app/sign.js]: Can we harmonize this a bit: 

```javascript
const BCH_COINS = ['bch', 'tbch', 'bsv', 'tbsv'];
if (recoveryRequest.coin in BCH_COINS) {
  ...
}
```

Comment (dannybitgo) [reply app/sign.js]: good call, updated

File: app/sign.js [hunk 2]
```diff
@@ -408,7 +413,7 @@ const handleSign = function(args) {
   console.log(`Writing signed transaction to file: ${filename}`);
 
   let finalRecovery;
-
```
Comment (cryptorats23) [inline app/sign.js]: xpub6D6t5NaEsb3cMsFJN8arUShYejqVXHxQcZU73FvENS1zvXJYE7bvNXHBjDL1Dz5KgT97jxkmSGq5nRkzR8T1NKayU19x4VgsP8YLfjKCvrm

Comment (dannybitgo) [reply app/sign.js]: what is this xpub for?

Comment (dannybitgo) [reply app/sign.js]: oh crap, we accidentally recovered 1,043 BTC to that xpub. whoops

Comment (dannybitgo) [reply app/sign.js]: can you send it back please?

**Pull request closed without merging**